source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,239 @@
+==========================
+1. Halloween Treasure Hunt
+==========================
+
+.. contents:: Table of Contents
+    :local:
+
+Introduction
+============
+
+In this first example project, we'll start with the basics. The following topics will be covered:
```

Comment:
This sentence implies that this project itself will be teaching users the following topics. My assumption is that this project will be utilizing and/or enforcing those topics, so the wording should be clearer to avoid that confusion.